modules/rtcpsummary/rtcpsummary.c

```diff
@@ -9,12 +9,15 @@
 
 
 static void print_rtcp_summary_line(const struct call *call,
-				    const struct stream *s)
+				    struct stream *s)
```

Comment:
is this module used ?

Perhaps we should move the xrtpstat code to a separate module, and allow it
to print the stats to a SIP header or stdout ?

Reply:
I have no idea if `rtcpsummary` is used.

Also have no idea how to make `call->sess` thread safe.
- `stream_rtcp_handler()` is called from RX thread,
-  while e.g. `call_hangup()` does a `mem_deref(call->sess)` in the main thread.

Another mutex in `struct call`?

@sreimers, @alfredh : The changes for thread-safety are already very large and complicated. Is there enough benefit for an RX real-time thread?

Or should we close this?

Is there a chance for a redesign of the RTP processing in order to have it thread-safe out of the box?